### ISSUE TITLE
ES modules shouldn't use `require`

### ISSUE THREAD

**Issue by lephyrus** [cartant/rxjs-marbles#49]
**The problem:**
I'd like to use rxjs-marbles for testing a library at work, but Rollup isn't able to bundle it.

**The reason:**
This is, as an example (it happens in many files), `rxjs-marbles/esm5/cases.js`:
```js
var __assign = (this && this.__assign) || Object.assign || function(t) {
    for (var s, i = 1, n = arguments.length; i < n; i++) {
        s = arguments[i];
        for (var p in s) if (Object.prototype.hasOwnProperty.call(s, p))
            t[p] = s[p];
    }
    return t;
};
var isArray = require("lodash/isArray");
export function _cases(adapter, cases) {
    if (!isArray(cases)) {
        cases = Object.keys(cases).map(function (key) { return (__assign({}, cases[key], { name: key })); });
    }
    cases.forEach(adapter);
}
```

This looks like a ES module, and is recognized as such, because of the `export` statements. But in the middle it also uses `require()` to pull in the lodash dependency. This mixture isn't any module format at all. It would require a bundler or runtime to apply both ESM and commonjs semantics to the same module. Rollup (sensibly, I think) does not do that.

**The solution:**
Unfortunately I don't know Webpack well enough to point to the source of that mixed module output.

**Comment by cartant** [comment]
This should be fixed in 4.3.2.

**Comment by lephyrus** [comment]
@cartant Wow, that was double-quick! I can confirm that it works now. Thank you very much.
